macros/NEW_2nuECEC.config.mac

```diff
@@ -0,0 +1,23 @@
+### --------------------------------------------------------
+###  This init macro simulates 2-neutrino ECEC decays
```

Comment:
I know I'm a bit of a pain, but could you use the same format as the other example macros :-)?

Reply:
You can't beat me at that ;). Do you mean like the one I just pushed?